docs/user_guide/bank/getting_started_choices.md

```diff
@@ -2,19 +2,18 @@ READY FOR REVIEW
 
 # Some Things to Think about before Getting Started
 
-Human Essentials is designed to help you manage your inventory and your relationship with partners, while also providing at least some information to help with your grant writing.
+Human Essentials is designed to help you manage your inventory and your relationship with partners, while also providing some information to help with your grant writing.
 
-Some questions that will help you when you are setting up Human Essentials.  This is not a complete list.
+Here are some questions that will help you when you are setting up Human Essentials. 
 
-- Do you (your bank) want to start out immediately with my partners making requests, or do you want to just track your distributions for awhile first?
-- Do you run on a monthly cycle, or more on an adhoc basis?
+- Do you (your bank) want to start out immediately with your partners making requests, or do you want to just track your distributions for awhile first?
```

Comment:
Tiniest nit…I _think_ it's `a while` when following a preposition and only `awhile` when used as an adverb.
```suggestion
- Do you (your bank) want to start out immediately with your partners making requests, or do you want to just track your distributions for a while first?
```

Reply:
For a while looks weird to me though   -- how about    

 Do you (your bank) want to start out immediately with your partners making requests, or do you want to just track your distributions awhile first?

Reply:
Looking in various usage guides,  it seems like for a while is technically correct, but for awhile is quite commonly used.   I'm just going to reword it.